demos/headless/headless.go

```diff
@@ -266,7 +267,7 @@ func (c *ScoreSystem) Update(entity *ecs.Entity, dt float32) {
 		c.upToDate = true
 		c.scoreLock.RUnlock()
 
-		render.Display = basicFont.Render(render.Label)
+		render.SetDrawable(basicFont.Render(render.Label))
```

Comment:
Because the `SetDrawable` function does two things:
* Set the variable value
* Update the OpenGL Buffers (i.e.: how big is this texture, what are the X,Y coords for each corner of the texture, etc.)

The latter is why there's a `SetDrawable` function, because forgetting to do that, will result in weird behavior.